### DISCUSSION TITLE
Starship

### DISCUSSION THREAD

**Discussion by sheldonhull** [JanDeDobbeleer/oh-my-posh#1293] [Troubleshoot]
I've used starship for a while. Couldn't find any quick mention other than acknowledgment for Starship so I'm not clear on the difference.

Is OhMyPosh providing full theming like OhMyZsh, prompt customization like starship, and/or plugin options? If there's a doc section on the scope intended for the project please feel free to point to it. 

I'm curious since it's not limited to pwsh prompt only now and I know Go. 
Cheers!

**Comment by lnu** [comment]
most of the answers are here: https://ohmyposh.dev/ (portability, segments, configuration, ...)  
From my experience, omp is way easier to setup than starship.

**Reply by JanDeDobbeleer** [reply]
That and we support more theming capabilities out of the box looking at colors and segment styles. We're behind on segments but ahead when it comes to the core functionality.

**Reply by lnu** [reply]
it would be nice to have a feedback from an active starship user.

**Reply by kid1412621** [reply]
How about the performance diff?

**Reply by JanDeDobbeleer** [reply]
@kid1412621 please elaborate, positive or negative?

**Reply by kid1412621** [reply]
Hi @JanDeDobbeleer, don't get me wrong, just want to know the performance difference between oh-my-posh and starship. I know it will be vary when comes to different scenarios. Wondering if anyone made a general test. That's it.

**Reply by JanDeDobbeleer** [reply]
@kid1412621 I did a long time ago, to make sure it was on par. Recently Joel Benett did a quick [test](https://twitter.com/jaykul/status/1520065654233214982?s=21&t=W5a5MQ3LR3LaEkcF_VWYsQ) and oh-my-posh was the fastest (out of the box experience). We also had recent reports via Twitter that oh-my-posh was faster than powerlevel10k, which is pretty crazy.

**Reply by kid1412621** [reply]
Cool

**Comment by saumyajyoti-mukherjee** [comment]
I prefer OMP over Starship. Main reason is I am using cmder/clink prompt. OhMyPosh uses clink async update feature whereas Starship does not. It makes a noticeable difference for large git repro git prompt updating. Starship is almost unusable for me in git folders as I have to wait for ~ 5 sec for each prompt update.
